gdsfactory/routing/route_bundle.py

```diff
@@ -260,7 +258,7 @@ def straight_dbu(
 
     if waypoints is not None and not isinstance(waypoints[0], kf.kdb.Point):
         _waypoints: list[kf.kdb.Point] | None = [
-            kf.kdb.Point(p[0] / dbu, p[1] / dbu) for p in waypoints
+            kf.kdb.Point(c.kcl.to_dbu(p[0]), c.kcl.to_dbu(p[1])) for p in waypoints
```

Comment:
Here you can create a DPoint first and call to_dbu on it. The function works on pretty much any klayout shape object as well